src/runtimeContext.ts

```diff
@@ -5,7 +5,16 @@ import { assert, hasOwn, warn } from './utils'
 let vueDependency: VueConstructor | undefined = undefined
 
 try {
-  vueDependency = require('vue')
+  const requiredVue = require('vue')
+  if (requiredVue && isVue(requiredVue)) {
+    vueDependency = requiredVue
+  } else if (
+    requiredVue &&
+    'deafult' in requiredVue &&
+    isVue(requiredVue.deafult)
+  ) {
+    vueDependency = requiredVue.deafult
```

Comment:
```suggestion
    'default' in requiredVue &&
    isVue(requiredVue.default)
  ) {
    vueDependency = requiredVue.default
```

Guess this is a typo.

Reply:
I must be too sleepy 😓